docs/project_overview.md

```diff
@@ -103,8 +103,27 @@ https://github.com/mickahell/qiskit-ecosystem_template.
 
 ### Storage
 
-We use json file ([members.json](https://github.com/qiskit-community/ecosystem/blob/main/ecosystem/resources/members.json)) as ecosystem state storage. 
-Access to members.json is handled through [JsonDAO](https://github.com/qiskit-community/ecosystem/blob/main/ecosystem/daos/jsondao.py) class.
+We store each member of the ecosystem as a TOML file under
+[`ecosystem/resources/members`](https://github.com/qiskit-community/ecosystem/blob/main/ecosystem/resources/members);
+these are the files you should edit when adding / updating members to the
+ecosystem. Access to this file is handled through the
+[`JsonDAO`](https://github.com/qiskit-community/ecosystem/blob/main/ecosystem/daos/jsondao.py)
+class.
+
+When `JsonDAO` writes updates, it will also compile a json file
+([`members.json`](https://github.com/qiskit-community/ecosystem/blob/main/ecosystem/resources/members.json))
+to be read by qiskit.org/ecosystem. If this file becomes out-of-sync with the
+TOML files, you can recompile it using
+
+```sh
+python -m manager recompile
+```
+
+You shouldn't edit `members.json` manually.
+
+If you have a merge conflict in `members.json`, don't try to manually resolve the
+conflict. Instead, merge `main` and use the version of `members.json` from `main`.
+Then, recompile the file with `python -m manager recompile`.
```

Comment:
DOn't we want to mention to push at all `members.json`as we don't want any manual generation ?

Reply:
Yeah I think you're right, I've moved the writing of `members.json` out of the database writing step in [99101aa](https://github.com/qiskit-community/ecosystem/pull/426/commits/99101aafcdfacddbe474c90d42d5d7b2933cc977). I think it should be compiled by an action (see [89913d4](https://github.com/qiskit-community/ecosystem/pull/426/commits/89913d43011264d6664d964d64669bd3ad4bde5b)).